### ISSUE TITLE
Мы: некоммерческое учреждение "Центр урбанистических проектов и инициатив "Прастора" (Витебск) - задача дададзена [5/15/2017 22:05:15]

### ISSUE THREAD

**Issue by falanster** [diglabby/tasks#14]
Коратка пра ініцыятыву: «ПРАСТОРА» создает возможность для всех горожан Витебска управлять своим городом через их участие в преобразовании городской среды.
СТРАТЕГИЧЕСКИЕ ЦЕЛИ:

ОТСТАИВАТЬ право горожан Витебска на жизнь в дружественной, комфортной, безопасной, привлекательной, доступной и экологичной городской среде.
ПРИВЛЕКАТЬ  творчески ориентированных людей города Витебска к участию в креативной экономику.
Патрэба ініцыятывы: Нам год и до этого мы в интернете были представлены в соцсетях @Prastora.Vitebsk в этом году мы решили создать сайт. Нам взялись помочь волонтеры из Витебска. Но они не делают дизайн. Наша просьба - это помочь создать дизайн сайта. У нас есть лого и фирменные цвета.

**Comment by svetit** [comment]
Падаецца, годная ініцыятыва :)
Але ж я думала, што там нейкая Кухня з Віцебска, як раз робіць дызайны, ну да ладна.
Можна пашукаць у нас.
Лагатып і колеры у іх цікавыя.

**Comment by falanster** [comment]
так, але варта улічваць, што нават для добрай ініцыятывы мы не сэрвіс...

**Comment by svetit** [comment]
Так.

**Comment by falanster** [comment]
@svetit

вось дарэчы, што яны прыслалі... чым не дазайн? =)))
![etxc0rgyc4q](https://cloud.githubusercontent.com/assets/1325280/26139899/4edbff8a-3adc-11e7-8706-342aa46029db.jpg)

**Comment by svetit** [comment]
Для пачатку дык увогулле окей :) 
Дык а што ім трэба звыш гэтага? Іншыя старонкі у гэтым жа стылі?

**Comment by falanster** [comment]
@svetit пішуць, што хочуць інфаграфіку якая будзе тлумачыць іх прынцыпы працы...
